st2api/tests/unit/controllers/v1/test_actions_rbac.py

```diff
@@ -0,0 +1,392 @@
+# Copyright 2020 The StackStorm Authors.
```

Comment:
We will have to move these tests to https://github.com/StackStorm/st2-rbac-backend/blob/master/tests/unit/controllers/api/v1/test_actions_rbac.py. You will need to open a separate PR at the st2-rbac-backend repo. Let's do this when the code is ready so we know the changes are working.